WBooks/Models/Comment.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Comment.swift
+//  WBooks
+//
+//  Created by ignacio.mendez on 12/08/2021.
+//
+
+import Foundation
+
+public struct Comment: Codable {
+    let id: Int
+    let user_id: Int
+    let book_id: Int
```

Comment:
Let's use **camelCase** for model properties. You can set coding keys to parse "user_id" to "userId".